modules/appeals_api/spec/workers/decision_review_report_daily_spec.rb

```diff
@@ -4,19 +4,20 @@
 
 describe AppealsApi::DecisionReviewReportDaily, type: :job do
   describe '#perform' do
+    recipients = %w[
```

Comment:
```suggestion
let(:recipients) do %w[
```

Best practice is to use `let`s within `describe` blocks rather than block variables.

Reply:
recipients was not shared, so moved it to inside the test it was used in